src/Maestro/Maestro.Web/Api/v2020_02_20/Controllers/BuildsController.cs

```diff
@@ -363,15 +365,16 @@ private async Task SetBuildIncoherencyInfoAsync(Data.Models.Build build)
                             Commit = incoherence.Commit
                         });
                     }
+                    build = await context.Builds.FindAsync(buildId);
+                    build.Incoherencies = incoherencies;
 
                     context.Builds.Update(build);
                     await context.SaveChangesAsync();
                 }
             }
             catch (Exception e)
             {
-                Logger.LogWarning(e, $"Problems computing the dependency incoherencies for a new build of " +
-                    $"{build.AzureDevOpsBuildNumber} from {(build.AzureDevOpsRepository ?? build.GitHubRepository)}");
+                Logger.LogWarning(e, $"Problems computing the dependency incoherencies for BAR build {buildId}");
```

Comment:
NIT: I preferred the old way

Reply:
Would sure be nice, but the exception might happen when getting the build.